### ISSUE TITLE
Prevent Global Duplicate Prevents Undo Last Action

### ISSUE THREAD

**Issue by kanjieater** [Renji-XD/texthooker-ui#17]
![image](https://user-images.githubusercontent.com/32607317/219800376-ea22faa6-4827-4163-a454-14639132de1b.png)

Steps to reproduce:
1. Add some text https://renji-xd.github.io/texthooker-ui/
2. Enable Prevent Global Duplicate
3. Delete a line
4. Undo deleting that line
5. The delete was not undone

**Comment by Renji-XD** [comment]
i think this is similar to your other issues a matter of definition and the way it was intended to work. when changing it i would probably add a new option to allow duplicates on undo or sth. similar

**Comment by kanjieater** [comment]
Yeah I think it is similar. I think you probably want to still be able to undo if you have global enabled though.

**Comment by Renji-XD** [comment]
i missunderstood the request. I thought you wanted to allow inserting duplicates on undo - that you can't in general undo lines with global duplicate check enabled was a bug
